### PR TITLE
Cherry picking iOS9 related fixes

### PR DESCRIPTION
**Pull request by wmathurin**
Last one included because it touched the same line as the second one:
wmathurin	More refined Application Transport Security settings	6f9597a
wmathurin	On iOS9 authentication required on every launch	a166b55
bhariharan	Using different NSFileManager instances	767d7c6

### CODE REVIEW

**Comment by wmathurin** [comment]
We will cut a patch in the coming weeks.

File: build/app_template_files/__NativeTemplateAppFiles__/__NativeTemplateAppName__/__NativeTemplateAppName__-Info.plist [hunk 1]
```diff
@@ -1,82 +1,71 @@
...
-            <key>NSExceptionRequiresForwardSecrecy</key>
-            <false/>
-          </dict>
+          <key>NSIncludesSubdomains</key>
+          <true/>
+          <key>NSExceptionRequiresForwardSecrecy</key>
+          <false/>
         </dict>
-        <!-/- -->
+        <key>force.com</key>
```
Comment (bhariharan) [inline build/app_template_files/__NativeTemplateAppFiles__/__NativeTemplateAppName__/__NativeTemplateAppName__-Info.plist]: Do we need ```salesforce.com``` too?

Comment (bhariharan) [inline build/app_template_files/__NativeTemplateAppFiles__/__NativeTemplateAppName__/__NativeTemplateAppName__-Info.plist]: Never mind. It's above. I just saw.

**Comment by bhariharan** [comment]
:+1:

**Pull request merged**